config/config.go

```diff
@@ -96,6 +98,7 @@ func init() {
 	cfg.SetDefault("analyzer.replication.debug", false)
 	cfg.SetDefault("analyzer.topology.backend", "memory")
 	cfg.SetDefault("analyzer.topology.probes", []string{})
+	cfg.SetDefault("analyzer.topology.k8s.config_file", "/etc/skydive/kubeconfig")
```

Comment:
you are missing a default for `analyzer.topology.istio.config_file`

Reply:
done